src/components/TransactionDetails/Transfer/Transfer.less

```diff
@@ -0,0 +1,55 @@
+@import "../../../views/resources/css/variables.less";
+.transaction-details-outer-container{
+    max-height: 2.8rem;
+    overflow-y: scroll;
+  }
+  .transaction-details-item-inner-container {
+    display              : grid;
+    grid-row-gap         : .1rem;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows   : 1fr 1fr 1fr;
```

Comment:
why introduce `fr` sizes ? Please use consistent sizing format

Reply:
sorry,I just want to know there is any problem about fr ?